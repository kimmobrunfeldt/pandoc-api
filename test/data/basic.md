# Basic test

Description.

![test](does-not-exist.png)

## Heading 2

### Heading 3

#### Heading 4

##### Heading 5
