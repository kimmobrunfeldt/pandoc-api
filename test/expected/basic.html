<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta http-equiv="Content-Style-Type" content="text/css" />
  <meta name="generator" content="pandoc" />
  <title></title>
  <style type="text/css">code{white-space: pre;}</style>
</head>
<body>
<h1 id="basic-test">Basic test</h1>
<p>Description.</p>
<div class="figure">
<img src="does-not-exist.png" alt="test" />
<p class="caption">test</p>
</div>
<h2 id="heading-2">Heading 2</h2>
<h3 id="heading-3">Heading 3</h3>
<h4 id="heading-4">Heading 4</h4>
<h5 id="heading-5">Heading 5</h5>
</body>
</html>
